test/LockedNORI.test.ts

```diff
@@ -178,9 +177,20 @@ const setupWithGrant = async (
     },
   } as TokenGrantOptions;
 
-  await expect(
-    lNori.createGrant(
-      grantAmount,
+  const userData = ethers.utils.defaultAbiCoder.encode(
```

Comment:
i'm seeing that I should probably create a utility for this grant creation using `batchCreateGrants` because it happens like 4 times in here